tslearn/neighbors.py

```diff
@@ -7,7 +7,11 @@
 from sklearn import neighbors
 from sklearn.neighbors import (KNeighborsClassifier, NearestNeighbors,
                                KNeighborsRegressor)
-from sklearn.neighbors.base import KNeighborsMixin
```

Comment:
Maybe this should be moved to the discussion in #211 ?

Reply:
Yes, I didn't think that we would create another issue just for this, but since it's already done I will revert this change.